docs/sdks/architecture/deployment-modes.md

```diff
@@ -142,3 +142,11 @@ A/B/n testing methodology relies on the assumption that we have a clear picture
 Both of these issues are in theory solvable if teams are diligent about filtering on the correct exposure event. However, as experimentation programs grow we have observed that this becomes increasingly hard to enforce and monitor. As you consider different options to deploy Eppo's SDK, make sure that you consider not just immediate implementation costs, but also long term scalability and tech debt. 
 
 For teams that need to fetch variants at session start (say, for fully server-side rendered architectures), Eppo can support this pattern. This is done by exporting a list of flags from the SDK and evaluating each of them in a for loop. That said, for the reasons above it is highly encouraged that teams instead follow one of the other deployment patterns whenever possible.
+
+## Precomputed Assignments
```

Comment:
Hoping these don't seem to forced/random. I'm trying to get more visibility in the docs for Precomputed Assignments

Reply:
Put this higher in the document, this is the 2nd desired deployment mode: after `Local flag evaluation using configurations from CDN (recommended)`